### PR TITLE
fix: missing participants in conference calls - FS-332

### PR DESCRIPTION
**Pull request by David-Henner**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/FS-332" title="FS-332" target="_blank">BugFS-332</a>  [iOS] Missing participants in federated conference calls
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

When connecting to a conference call, only the self participant is present in the call, even though other participants have answered.

### Causes (Optional)

When adding federation support, the uuids provided to avs were serialised using the `uuidString` property of `UUID`, which generates an uppercased string. While, on avs side, they lowercase the identifiers received in `set_clients_for_conv`. It then creates a mismatch (lowercase vs uppercase) between the clients identifiers stored by avs and the identifiers provided through the other avs APIs.

### Solutions

Revert back to serialising the UUIDs using `transportString()` instead of `uuidString`

### Testing

Verified that `AVSIdentifier` serialises the UUIDs using lowercase

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [x] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [x] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Tests/Source/Calling/AVSIdentifierTests.swift [hunk 1]
```diff
@@ -49,6 +49,15 @@ class AVSIdentifierTests: XCTestCase {
...
     }
 
+    func testThatItSerializesUUIDToLowercase() {
+        // When
+        let uuid = UUID()
+        let sut = AVSIdentifier(identifier: uuid, domain: nil)
+
+        // Then
+        XCTAssertEqual(sut.serialized, uuid.transportString())
+    }
```
Comment (johnxnguyen) [inline Tests/Source/Calling/AVSIdentifierTests.swift:59]: This doesn't explicitly assert that it's lowercase, just that it's the same as `.transportString()`.

Comment (David-Henner) [reply Tests/Source/Calling/AVSIdentifierTests.swift]: good point 👍

**Review by johnxnguyen — APPROVED** [review]

**Comment by David-Henner** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @David-Henner! Merging PR for command **release patch** for PR #**1643**. Will create a **patch** release after merged.
Next version is 411.0.2.

**Pull request merged**